README.md

```diff
@@ -6,3 +6,9 @@ The AMP Advisory Committee (AC) provides perspective and advice to the Technical
 - [Our working mode](https://github.com/ampproject/meta-ac/blob/master/WORKING_MODE.md)
 - [Meeting minutes](https://github.com/ampproject/meta-ac/tree/master/meetings)
 - [Upcoming meetings](https://github.com/ampproject/meta-ac/labels/AC%20Meeting)
+
+## Communication Channels
+
+- [Slack](https://docs.google.com/forms/d/e/1FAIpQLSd83J2IZA6cdR6jPwABGsJE8YL4pkypAbKMGgUZZriU7Qu6Tg/viewform?fbzx=4406980310789882877) [#amp-ac](https://amphtml.slack.com/messages/amp-ac/) - Public
```

Comment:
Can you make the link to the form a bit more explicit? e.g. "To get invited to the Slack workspace, add your email to this form."

Reply:
Addressed in 7987f08